instructions.md

```diff
@@ -0,0 +1,42 @@
+* **Visualize location uncertainties** toggles visualizing the uncertainties in location for each earthquake marker on and off. If the catalog contains no information about location uncertainties, a default range of 500 m is used. The uncertainties are related to the location of the hypocenter, the size of the marker is not taken into account.
+* **Show fault lines** toggles the Southern California fault lines on and off. Only available for the two SCEDC catalogs. **NB**: slows down the application, not recommended with the play-feature.
+
+The time slider below the map can be either moved manually, or played automatically from start to end. Using the play-feature will automatically move the time slider one step at a time, once every three seconds, until the end. The currently shown time step with start and end times is visible underneath the slider. Clicking on an earthquake on the map will display all the information related to it.
+
+##### Scatter plot
+
+The configurations available on the scatter plot are
+*  **Time range** allows the user to select the start and the end dates for the earthquakes shown on the plot. All the earthquakes that occurred between the start and the end dates are shown.
+* **X/Y-axis** settings define which column is used for which of the axes. Any column, even non-numerical ones can be selected here.
+* **Size column** is used to define which column from the uploaded data is used to set the size for the marker of each earthquake. Only numerical columns are listed as valid choices.
+* **Color column** defines the column used for setting the color of each marker. Only numerical columns are listed as valid choices.
+
+The plot can be zoomed into by selecting an area to zoom into or by clicking on the zoom icons in the top right corner of the plot. Zooming out can also be done by a double-click. After moving the plot around, it can be restored to its original position and zoom level by clicking on the "Reset axes" icon. Other options include printing a png of the plot, and selecting what is shown when hovering over a marker.
+
+##### Clustering
+
+The clustering method used is based on a distance measure introduced by [Zaliapin and Be-Zion (2013)](https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1002/jgrb.50179), which defines the distance between two earthquakes in terms of the difference in their location, time of occurrence, and magnitude. The distances are then used to find clusters with pre-, main, and aftershocks, similarly to an accompanying article [Zaliapin and Be-Zion (2013)](https://agupubs.onlinelibrary.wiley.com/doi/epdf/10.1002/jgrb.50178).
```

Comment:
I would change this to:
"... The distances are then used to find clusters with fore-, main-, and aftershocks, similarly to ..."

Reply:
Otherwise, this looks great!